### PR TITLE
fix: fix wrong index for dataSource in AllSummary widget & feat: add tooltip for AllSummary widget 

### PR DESCRIPTION
**Pull request by yuda110**
### To Reviewers
- [ ] Skip (`style`, `chore`, `ci` ONLY)
- [x] Not that difficult

### Type of Change
- [ ] New feature
- [x] Bug fixes
- [x] Feature improvement
- [ ] Refactor
- [ ] Others (performance improvement, CI/CD, etc.)

### Affects to
- [ ] Packages
  - [ ] core-lib
  - [ ] mirinae
  - [ ] etc 

- [ ] Apps
  - [ ] storybook
  - [x] web

### Checklist
- Did you check the `lint` and `type`?
- Did you document the changes?
  - Changes in `mirinae` should be reflected in `storybook`.
- Did you test the app after the package changes?

### Description
SSIA

### CODE REVIEW

File: apps/web/src/services/home-dashboard/modules/all-summary/AllSummary.vue [hunk 1]
```diff
@@ -195,6 +203,11 @@ const drawChart = () => {
     columnSeries.data.setAll(cloneDeep(chartState.data));
     chart.series.push(columnSeries);
 
+    // create tooltip
+    const tooltip = chartHelper.createTooltip();
+    chartHelper.setXYSingleTooltipText(chart, tooltip);
+    columnSeries.set('tooltip', tooltip);
```
Comment (yuda110) [inline apps/web/src/services/home-dashboard/modules/all-summary/AllSummary.vue:209]: 스크린샷 2023-09-22 오전 10 05 58

File: apps/web/src/services/home-dashboard/modules/all-summary/AllSummary.vue [hunk 2]
```diff
@@ -414,10 +419,6 @@ watch(() => state.selectedDateType, async () => {
 watch(() => state.dataSourceId, (dataSourceId) => {
     if (dataSourceId) getBillingCount();
 }, { immediate: true });
-
-// onUnmounted(() => {
-//     if (state.chart) state.chart.dispose();
-// });
```
Comment (piggggggggy) [inline apps/web/src/services/home-dashboard/modules/all-summary/AllSummary.vue:420]: Question: then when chart is disposed?

**Review by piggggggggy — APPROVED** [review]
LGTM!

**Review by skdud4659 — APPROVED** [review]
LGTM!

File: apps/web/src/services/home-dashboard/modules/all-summary/AllSummary.vue (hunk 2 above)
Comment (yuda110) [reply apps/web/src/services/home-dashboard/modules/all-summary/AllSummary.vue:420]: Oh Thanks, I'll add it

**Pull request merged**